src/components/Newbeer.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react'
+import Navbar from './Navbar'
+import axios from 'axios';
+
+function NewBeer() {
+    const [name, setName] = useState('');
+    const [tagline, setTagline] = useState('');
+    const [description, setDescription] = useState('');
+    const [firstBrewed, setFirstBrewed] = useState('');
+    const [attenuationLevel, setAttenuationLevel] = useState(0);
+    const [contributedBy, setContributedBy] = useState('');
+
+    const handleName = (e) => setName(e.target.value);
+    const handleTagline = (e) => setTagline(e.target.value);
+    const handleDescription = (e) => setDescription(e.target.value);
+    const handleFirstBrewed = (e) => setFirstBrewed(e.target.value);
+    const handleAttenuationLevel = (e) => setAttenuationLevel(e.target.value);
+    const handleContributedBy = (e) => setContributedBy(e.target.value);
+
+    const handleNewBeer = async (e) => {
+        e.preventDefault();
+        try {
+            const body = { name, tagline, description, firstBrewed, attenuationLevel, contributedBy };
+
+            await axios.post('https://ih-beers-api2.herokuapp.com/beers/NewBeer', body);
```

Comment:
Here the url you need to use to create a beer is different, the correct url is:
`"https://ih-beers-api2.herokuapp.com/beers/new"`